tests/webgl_sample_query.cpp

```diff
@@ -37,9 +38,7 @@ void getQueryResult()
 
   GL_CALL(glDeleteQueries(1, &sampleQuery));
 
-#ifdef REPORT_RESULT
-  REPORT_RESULT(result);
-#endif
+  exit(result);
```

Comment:
Is this `result` expected to be 0 when the test succeeds? The same question for webgl_timer_query.cpp.

Reply:
One needs to look at the test_browser.py to see what the expected return code of the test is:

```
def test_webgl_timer_query(self):
  ...
  self.btest_exit('webgl_timer_query.cpp', args=cmd)
```

In this case we see no special `assert_returncode` param so the expected result here is zero.

Should I change this to?:

```
assert(result == 0);
exit(0);
```

Or maybe:

```
exit(result == 0 ? EXIT_SUCCESS : EXIT_FAILURE);
```

There a several different ways we could express it I guess, I'm not sure which is the most clear.   Probably the one with the assert?

Reply:
Actually I quite like the current version because it keeps this change obvious in that it converted `REPORT_RESULT(XXX)` directly to `exit(XXX)`.    Kind of nice one to one mapping and it means the expectation in the python code stays the same.